.circleci/config.yml

```diff
@@ -211,18 +220,20 @@ on-integ-and-version-tags: &on-integ-and-version-tags
     tags:
       only: /^v[0-9].*/
 
+#----------------------------------------------------------------------------------------------------------------------------------
 
 workflows:
   version: 2
   default-flow:
     jobs:
       - build:
-          <<: *on-any-branch
+          <<: *not-on-integ-branch
       - platforms-build:
           context: common
+          #@@<<: *on-integ-and-version-tags
           <<: *on-any-branch
           matrix:
             parameters:
-              platform: [focal, bionic, xenial, centos7]
+              platform: [focal, bionic, xenial, centos8, centos7, buster]
```

Comment:
IMHO we should remove all these random OSes. There isn't a behavioural difference between them. I realize that in the past we **wanted** this, but I think rust has proven stable enough.